server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java

```diff
@@ -30,11 +31,14 @@ public class TableManipulation implements AutoCloseable {
    private static final String ID_COLUMN_NAME = "id";
    private static final String DEFAULT_IDENTIFIER_QUOTE_STRING = "\"";
    private static final String TABLE_NAME_PREFIX = "tbl";
-   private final String tableName;
+   private String tableName;
 
    public TableManipulation(String cacheName, PooledConnectionFactoryConfigurationBuilder persistenceConfiguration, ConfigurationBuilder configurationBuilder) {
       this.persistenceConfiguration = persistenceConfiguration;
       this.tableName = String.format("%s%s_%s%s", DEFAULT_IDENTIFIER_QUOTE_STRING, TABLE_NAME_PREFIX, cacheName, DEFAULT_IDENTIFIER_QUOTE_STRING);
+      if (isMysql()) {
+         this.tableName = this.tableName.replaceAll("\"", "");
```

Comment:
In our production code we use ` as the identifier quote string, shouldn't it be the same here?

Reply:
@ryanemerson AbstractTableManager use the same " as the identifier quote string

Reply:
MySQL uses the `MyTableOperations` implementation which has the ` identifier